test/http/echo.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2023 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package http
+
+import (
+	"fmt"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/nuts-foundation/nuts-node/core"
+	"github.com/nuts-foundation/nuts-node/test"
+	"net/http"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func StartEchoServer(t *testing.T, registerRoutesFunc func(router core.EchoRouter)) string {
```

Comment:
This test http server uses echo as test server instead of httptest. Isn't this abusing the fact echo is loaded as library? (replacing our main http server would then also start influencing the test http server setup)

Reply:
Removed